zeq/hbp/detail/imageRawRGBA8.fbs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015, Human Brain Project
+//                     Grigori Chevtchenko <grigori.chevtchenko@epfl.ch>
+
+namespace zeq.hbp;
+
+table ImageRawRGBA8
```

Comment:
Do we need Raw? Other images would be called ImageJPEG, so ImageRGBA8 is enough for me.